services/portal/studyViewer/studyViewerTasks.js

```diff
@@ -0,0 +1,76 @@
+const studyViewerProps = require('./studyViewerProps.js');
+const users = require('../../../utils/user');
+
+const I = actor();
+
+module.exports = {
+
+  goToStudyViewerPage() {
+    I.amOnPage(studyViewerProps.path); /// study-viewer/clinical_trials
+    I.waitForVisible(studyViewerProps.studyViewerDivClass, 5); // .study-viewer
+    I.saveScreenshot('study_viewer_page.png');
+  },
+
+  goToStudyPage() {
+    // /study-viewer/clinical_trials/ACTT
+    I.amOnPage(studyViewerProps.datasetPath);
+    I.waitForVisible(studyViewerProps.datasetDivClass, 5);
+    I.saveScreenshot('dataset_page.png');
+  },
+
+  async clickRequestAccess() {
+    await I.seeElement(studyViewerProps.requestAccessButtonXPath);
+    I.click(studyViewerProps.requestAccessButtonXPath);
+  },
+
+  async loginToRequestAccess() {
+    await I.seeElement(studyViewerProps.loginRAButtonXPath);
+    I.click(studyViewerProps.loginRAButtonXPath);
+  },
+
+  async clickDownload() {
+    await I.seeElement(studyViewerProps.downloadButtonXPath);
+    I.click(studyViewerProps.downloadButtonXPath);
```

Comment:
there should be more steps to this - clicking the button opens a modal but you need to click again to download